### PR TITLE
cdf bugs 

### PR DESCRIPTION
**Pull request by kiante-fernandez**
bugs in DDM cdf addressed from Issue #23 Large negative cdf for values near tau.

### CODE REVIEW

File: src/DDM.jl [hunk 1]
```diff
@@ -175,15 +175,15 @@ function cdf(d::DDM{T}, t::Real; ϵ::Real = 1.0e-12) where {T<:Real}
         return T(NaN)
     end
 
-    K_s = _K_small(d, t; ϵ)
-    K_l = _K_large(d, t; ϵ)
+    K_l = _K_large(d, t; ϵ=ϵ)
```
Comment (itsdfish) [inline src/DDM.jl:178]: Is there a reason to use `ϵ=ϵ`? I recommend using just ϵ because it is simpler.

File: src/DDM.jl [hunk 2]
```diff
@@ -381,67 +383,325 @@ sampler(rng::AbstractRNG, d::DDM) = rand(rng::AbstractRNG, d::DDM)
 # """
 #     RatcliffDDM
```
Comment (itsdfish) [inline src/DDM.jl:384]: Given how long this file is, would you be willing to move this commented code into a separate file?

File: src/DDM.jl [hunk 3]
```diff
@@ -381,67 +383,325 @@ sampler(rng::AbstractRNG, d::DDM) = rand(rng::AbstractRNG, d::DDM)
...
+# loglike = logpdf.(dist, choice, rt)
+# ````
+    
+# # References
+        
+# Ratcliff, R., & McKoon, G. (2008). The Diffusion Decision Model: Theory and Data for Two-Choice Decision Tasks. Neural Computation, 20(4), 873–922.
+
+# Ratcliff, R. (1978). A theory of memory retrieval. Psychological Review, 85, 59–108. https://doi.org/10.1037/0033-295X.85.2.59
+
+# """
```
Comment (itsdfish) [inline src/DDM.jl:566]: FYI, you can remove the @concrete since you are specifying parametric types.

**Comment by itsdfish** [comment]
@kiante-fernandez, thank you for the PR. I had a few minor comments. Once those are addressed, I will merge it into the master branch.

File: src/DDM.jl (hunk 2 above)
Comment (kiante-fernandez) [reply src/DDM.jl:384]: I will create a new file called: `RatcliffDDM .jl`

File: src/DDM.jl (hunk 3 above)
Comment (kiante-fernandez) [reply src/DDM.jl:566]: Thanks!

**Pull request merged**